ext/odbc/tests/odbc_connect_001.phpt

```diff
@@ -16,35 +16,40 @@ include 'skipif.inc';
 include 'config.inc';
 $dsn = str_replace(";uid={$user};pwd={$pass}", '', $dsn);
 
+/*
+ * A bug in msodbcsql causes a memory leak when reconnecting after closing.
```

Comment:
xfail with link to a bug tracker might be better

Reply:
I also wouldn't comment all the close statements etc. We have xleak to handle this specifically. I'd advise to keep the test running when not under ASAN and use xleak + a link to the bugtracker.

Reply:
@nielsdos @mvorisek 
Thank you. 

This memory leak is not detected in normal debug mode and requires the following option:
https://github.com/php/php-src/blob/4022e287b20c310761d15efb64fb14aa080b18ed/.github/workflows/nightly.yml#L352

Will XLEAK still work in this case? In my Local environment, the test was treated as Failed even if I specified XLEAK.